gradle.properties

```diff
@@ -109,7 +109,7 @@ apacheFortressVersion=2.0.0
 apacheVelocityVersion=2.0
 apacheSyncopeVersion=2.0.8
 
-springBootVersion=1.5.16.RELEASE
+springBootVersion=1.5.17.RELEASE
```

Comment:
This is rather risky, because it impacts overlays. @hdeadman see any issues with this?